test/Crate/Test/DBAL/Functional/BindingTest.php

```diff
@@ -97,7 +97,7 @@ public function testBindTimestamp()
             $this->execute("DROP TABLE foo");
         }
 
-        $this->execute("CREATE TABLE foo (id int, ts timestamp) with (number_of_replicas=0)");
+        $this->execute("CREATE TABLE foo (id int, ts timestamp, PRIMARY KEY(id)) with (number_of_replicas=0)");
```

Comment:
I an not sure why the primary key is requires for this schema and the test. Could you elaborate?

Reply:
This change was made after this build failed: https://travis-ci.org/crate/crate-dbal/jobs/380583346. 

When the test `SchemaManagerTest::testListTables` runs, all tables are listed and the `information_schema.table_constraints` is queried for each table. 

It seems that this `foo` table has previously been created (and isn't dropped afterward). It doesn't have an entry in `information_schema.table_constraints` because it doesn't have a `PRIMARY KEY` constraint. 

I'm actually going to update this though to drop the `foo` table after the above test is run, because I think that's cleaner than modifying it like this.

Reply:
Fixed it here: 9229378